pkg/cloudprovider/gce/gce.go

```diff
@@ -309,6 +309,7 @@ func translateAffinityType(affinityType api.AffinityType) GCEAffinityType {
 // CreateTCPLoadBalancer is an implementation of TCPLoadBalancer.CreateTCPLoadBalancer.
 // TODO(a-robinson): Don't just ignore specified IP addresses. Check if they're
 // owned by the project and available to be used, and use them if they are.
+// TODO: Verify this is idempotent
```

Comment:
No, Justin was right, this is idempotent in that it assumes an already existing resource is a success - see the handling of http.StatusConflict errors.

You can get rid of the TODO though, unless your intent is for us to add automated verification.